pysurfex/configuration.py

```diff
@@ -550,7 +550,7 @@ def __init__(self, env, conf):
             gsize = float(env["LGSIZE"])
         trunc = 2  # linear
         if "TRUNC" in env:
-            trunc = int(env["TRUNC"])
+            trunc = int(float(env["TRUNC"]))
```

Comment:
Can you remove the int? It should be treated as a float if it's a float. If you convert it to an int the 2.4 value will be truncated to 2. Otherwise I support the change as I have had to do it also myself sometimes recently:-)

Reply:
Of course. I was not sure whether TRUNC should be "allowed" to actually be 2.4 due to the initial int constrain in pysurfex.